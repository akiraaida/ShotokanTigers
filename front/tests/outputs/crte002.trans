10 admin                00000 00000.00 A 
05 John Doe             00000 00100.00   
00                      00000 00000.00   
