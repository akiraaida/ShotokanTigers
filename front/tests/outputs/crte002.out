Please enter a command: 
Please enter your session type: 
Please enter a command: 
Please enter the account holder's name: 
Please enter an initial balance: 
Please enter a command: 
